grafana-init/README.md

```diff
@@ -42,6 +42,7 @@ Configuration
 | `GRAFANA_URL`      | `http://grafana:3000`  | Location of Grafana server      |
 | `GRAFANA_USERNAME` | `mini-mon`             | Agent Keystone username         |
 | `GRAFANA_PASSWORD` | `password`             | Agent Keystone password         |
+| `GRAFANA_USERS`    | `[{'user': GRAFANA_USERNAME, 'password': GRAFANA_PASSWORD, 'email': ''}]` | Agent Keystone users. Default datasource and dashboard are created for multiple users if this variable is set as, e.g. `"[{'user': 'mini-mon', 'password': 'password', 'email': ''}, {'user': 'username', 'password': 'password', 'email': ''}]"`. Default value is overwritten by `GRAFANA_USERNAME` and `GRAFANA_PASSWORD`.  **NOTE: Set this variable in string type** |
```

Comment:
You are expecting JSON from env variable so everything need to use double quotes `"`. You can check JSON validity on https://jsonformatter.curiousconcept.com/

Reply:
Done. Thanks for you comment.

Reply:
Sorry, I did not wrote it enough clearly:
JSON need all strings to be enclosed in double quotes:
```
{'username':'dfdsfdsf'} <= bad JSON
{"username":"dfdsfdsf"} <= good JSON
```
https://stackoverflow.com/a/4162651/1722542

You could try it with adding this to `docker-compose.yml` in `grafana-init` part
```
    environment:
      GRAFANA_USERS: '[{"user": "mini-mon2", "password": "password", "email": ""}, {"user": "username", "password": "password", "email": ""}]'
```

Reply:
I clearly understand.
Please check my change, thanks in advance.